### PR TITLE
Add english localization of guide

### PR DESCRIPTION
**Pull request by khramtsoff**

### CODE REVIEW

File: GUIDE-rus.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* *Response* - структура для передачи данных из Interactor в Presenter
+* *ViewModel* - структура для передачи данных из Presenter во ViewController
+Структуры Request, Response, ViewModel, объявляются внутри enum с наименованием модуля для выделенного пространства имен. DataFlow должен четко описывать бизнес задачи и давать наглядную информацию о передаваемых данных внутри модуля.
+
+## Interactor
+Содержит бизнес логику модуля. Нежелательно заставлять Interator ходить в сеть, обращаться к базе данных и т.д. Он выступает агрегатором бизнес логики. Так, например, для получения данных он обращается к Provider, к различным Worker'ам для выполнения каких-либо действий с данными, после чего передает подготовленные данные дальше в Presenter и т.д.
+Interactor получает запрос на операцию над данными от ViewController с помощью Request и предает данные в Presenter с помощью Response.
+
+## Worker
+Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
```
Comment (aeternas) [inline GUIDE-rus.md]: `фильтрации данных`

File: GUIDE.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
-# Компоненты YARCH
+# YARCH components
 
 ## Builder
-Отвечает за сборку модуля. В нем создаются необходимые сущности и проставляются зависимости.
+It is responsible for the assembly of the module. It creates the necessary entities and shows dependencies.
```
Comment (aeternas) [inline GUIDE.md]: `Responsible for the assembly of the module` ?

File: GUIDE.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
 
 ## DataFlow
-Представляет собой описание объектов для передачи данных (DTO - Data Transfer Objects) в рамках одного варианта использования (UseCase). Состоит из:
-* *Request* - структура для передачи данных из ViewController в Interactor
-* *Response* - структура для передачи данных из Interactor в Presenter
-* *ViewModel* - структура для передачи данных из Presenter во ViewController
-Структуры Request, Response, ViewModel, объявляются внутри enum с наименованием модуля для выделенного пространства имен. DataFlow должен четко описывать бизнес задачи и давать наглядную информацию о передаваемых данных внутри модуля.
+It is a description of objects for data transfer (DTO - Data Transfer Objects) within one use case (UseCase). Consists of:
```
Comment (aeternas) [inline GUIDE.md]: `Description`?

File: GUIDE.md [hunk 4]
```diff
@@ -1,50 +1,50 @@
...
+* * ViewModel * - structure for transferring data from Presenter to ViewController
+The Request, Response, ViewModel structures are declared inside the enum with the module name for the allocated namespace. DataFlow should clearly describe the business tasks and provide visual information about the data transmitted within the module.
 
 ## Interactor
-Содержит бизнес логику модуля. Нежелательно заставлять Interator ходить в сеть, обращаться к базе данных и т.д. Он выступает агрегатором бизнес логики. Так, например, для получения данных он обращается к Provider, к различным Worker'ам для выполнения каких-либо действий с данными, после чего передает подготовленные данные дальше в Presenter и т.д.
-Interactor получает запрос на операцию над данными от ViewController с помощью Request и предает данные в Presenter с помощью Response.
+It contains the business logic of the module. It is not advisable to force the Interator to go to the network, access the database, etc. He acts as an aggregator of business logic. So, for example, to get data, it calls the Provider, to various Workers to perform any actions with the data, and then sends the prepared data further to the Presenter, etc.
```
Comment (aeternas) [inline GUIDE.md]: `Contains...`

Comment (aeternas) [inline GUIDE.md]: `Interactor should not make network requests, access the database...`

Comment (aeternas) [inline GUIDE.md]: `It acts as an aggregator`

Comment (aeternas) [inline GUIDE.md]: `to perform any data actions`

File: GUIDE.md [hunk 5]
```diff
@@ -1,50 +1,50 @@
...
+Interactor receives a request for an operation on data from the ViewController with the Request and submits the data to the Presenter using Response.
```
Comment (aeternas) [inline GUIDE.md]: `data operation`

File: GUIDE.md [hunk 6]
```diff
@@ -1,50 +1,50 @@
...
 
 ## Worker
-Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
+Implements the reused part of the business logic (for example: algorithm for data filtering).
 
 ## Presenter
-Содержит логику подготовки данных для отображения (например: форматирование данных (номера телефона, суммы), локализация и т.д.) и передает их во ViewController посредством ViewModel.
+It contains the logic for preparing data for display (for example: formatting data (phone number, sum), localization, etc.) and passes them to ViewController through ViewModel.
```
Comment (aeternas) [inline GUIDE.md]: `data formatting ... passes it`
наверное вместо `through` можно `via`? Некритично

File: GUIDE.md [hunk 7]
```diff
@@ -1,50 +1,50 @@
...
 
 ## ViewController
-Отвечает за отображение состояния, полученого от Presenter и взаимодействует с пользователем. ViewController содержит в себе только одну view, которая создается в методе loadView(). Он не управляет расположением view. Вся необходимая верстка уже должна быть в отдельно реализованной view. Однако ViewController выступает делегатом для всего view слоя. Так он знает, когда изменился текст в TextField или пользователь нажал на ячейку.
+It is responsible for displaying the status received from Presenter and interacting with the user. ViewController contains only one view, which is created in the loadView () method. It does not control the location of the view. All necessary layout should already be in a separately implemented view. However, ViewController acts as a delegate for the entire view layer. So he knows when the Text in TextField changed or the user clicked on the cell.
```
Comment (aeternas) [inline GUIDE.md]: `All necessary layout should be already presented ... So it knows when the Text in TextField is/was changed ...`

File: GUIDE.md [hunk 8]
```diff
@@ -1,50 +1,50 @@
...
+    static let shared = CatalogDataStore ()
 
-    var catalogModels: [CatalogObjectType: [CatalogModel]] = [:]
+    var catalogModels: [CatalogObjectType: [CatalogModel]] = [:]
 }
 ```
 
 ## ViewModel
-Содержит в себе полностью подготовленные для отображения данные для View (каждой отдельной subview). ViewModel — это DTO и не содержит в себе никакой логики.
+Contains completely prepared for display data for View (each separate subview). ViewModel is a DTO and does not contain any logic.
```
Comment (aeternas) [inline GUIDE.md]: Лучше просто `prepared` наверное

File: GUIDE.md [hunk 9]
```diff
@@ -1,50 +1,50 @@
...
 
-# Передача данных между модулями
-Для передачи данных между модулями A и B требуется указать id передаваемой модели. Модуль B должен сам обратиться к общему хранилищу и получить требуемую модель. Нежелательно передавать данные из одного модуля в другой напрямую.
+# Data transfer between modules
+To transfer data between modules A and B, you must specify the id of the model to be transmitted. Module B should access the shared storage itself and obtain the required model. It is not desirable to transfer data from one module to another directly.
```
Comment (aeternas) [inline GUIDE.md]: `You should specify the Id of the model to be transmitted in order to transfer data between modules A and B ...  It is not required to directly transfer data from one module to another`, кажется directly так стоять должно

**Review by aeternas — CHANGES_REQUESTED** [review]
Посмотри пожалуйста мои рекомендации

File: GUIDE.md (hunk 9 above)
Comment (khramtsoff) [reply GUIDE.md]: кажется между "Нежелательно" и "It is not required" есть разница; оставил регистр "id"; остальное поправил

File: GUIDE.md (hunk 8 above)
Comment (khramtsoff) [reply GUIDE.md]: "полностью подготовленные" и "completely prepared", с точки зрения перевода норм, если дело в содержании, то лучше отдельным PR сразу в два языка

File: GUIDE.md (hunk 6 above)
Comment (khramtsoff) [reply GUIDE.md]: data formatting, via - поправил; их/them - кажется норм

File: GUIDE.md (hunk 5 above)
Comment (khramtsoff) [reply GUIDE.md]: >получает запрос на операцию над данными/receives a request for an operation on data
я бы оставил

**Review by xzeror — COMMENTED** [review]
Looks good to be merged.

**Review by aeternas — DISMISSED** [review]

File: GUIDE.md [hunk 10]
```diff
@@ -1,50 +1,50 @@
-# Компоненты YARCH
+# YARCH components
 
 ## Builder
-Отвечает за сборку модуля. В нем создаются необходимые сущности и проставляются зависимости.
+Responsible for the assembly of the module. It creates the necessary entities and shows dependencies.
```
Comment (iltsarev) [inline GUIDE.md]: Builder is responsible for the assembly of the module. It creates all necessary entities and shows dependencies.

File: GUIDE.md [hunk 11]
```diff
@@ -1,50 +1,50 @@
...
+* * ViewModel * - structure for transferring data from Presenter to ViewController
+The Request, Response, ViewModel structures are declared inside the enum with the module name for the allocated namespace. DataFlow should clearly describe the business tasks and provide visual information about the data transmitted within the module.
 
 ## Interactor
-Содержит бизнес логику модуля. Нежелательно заставлять Interator ходить в сеть, обращаться к базе данных и т.д. Он выступает агрегатором бизнес логики. Так, например, для получения данных он обращается к Provider, к различным Worker'ам для выполнения каких-либо действий с данными, после чего передает подготовленные данные дальше в Presenter и т.д.
-Interactor получает запрос на операцию над данными от ViewController с помощью Request и предает данные в Presenter с помощью Response.
+Contains the business logic of the module. Interactor should not make network requests, access the database, etc. It acts as an aggregator of business logic. So, for example, to get data, it calls the Provider, to various Workers to perform any data actions, and then sends the prepared data further to the Presenter, etc.
```
Comment (iltsarev) [inline GUIDE.md]: Interactor contains the business logic of the module. It shouldn't make any network requests, access the database, etc.

Comment (iltsarev) [inline GUIDE.md]: So, for example, to get data, it calls the Provider, to various Workers to perform any data actions, and then sends the prepared data further to the Presenter, etc. 

-> 

For example it calls the Provider to get data, various Workers to perform any data actions, etc.  When the data occurred, Interactor sends it to the Presenter.

File: GUIDE.md [hunk 12]
```diff
@@ -1,50 +1,50 @@
...
+Interactor receives a request for an operation on data from the ViewController with the Request and submits the data to the Presenter using Response.
 
 ## Worker
-Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
+Implements the reused part of the business logic (for example: algorithm for data filtering).
```
Comment (iltsarev) [inline GUIDE.md]: Worker should implement business logic that can be reused in another module (for example: algorithm for data filtering).

File: GUIDE.md [hunk 13]
```diff
@@ -1,50 +1,50 @@
...
 
 ## Presenter
-Содержит логику подготовки данных для отображения (например: форматирование данных (номера телефона, суммы), локализация и т.д.) и передает их во ViewController посредством ViewModel.
+Contains the logic for preparing data for display (for example: data formatting (phone number, sum), localization, etc.) and passes them to ViewController via ViewModel.
```
Comment (iltsarev) [inline GUIDE.md]: Presenter contains

Comment (iltsarev) [inline GUIDE.md]: and passes **it** to ViewController via ViewModel.

File: GUIDE.md [hunk 14]
```diff
@@ -1,50 +1,50 @@
...
 
 ## ViewController
-Отвечает за отображение состояния, полученого от Presenter и взаимодействует с пользователем. ViewController содержит в себе только одну view, которая создается в методе loadView(). Он не управляет расположением view. Вся необходимая верстка уже должна быть в отдельно реализованной view. Однако ViewController выступает делегатом для всего view слоя. Так он знает, когда изменился текст в TextField или пользователь нажал на ячейку.
+Responsible for displaying the status received from Presenter and interacting with the user. ViewController contains only one view, which is created in the loadView () method. It does not control the location of the view. All necessary layout should be already presented in a separately implemented view. However, ViewController acts as a delegate for the entire view layer. So it knows when the Text in TextField was changed or the user clicked on the cell.
```
Comment (iltsarev) [inline GUIDE.md]: ViewController is responsible the ViewModel received from Presenter

Comment (iltsarev) [inline GUIDE.md]: It doesn't control the location of the view and it's layout

Comment (iltsarev) [inline GUIDE.md]: All necessary layout should be configured in a separately implemented view.

Comment (iltsarev) [inline GUIDE.md]: It knows when the text in UITextField was changed or the user clicked on the cell.

File: GUIDE.md [hunk 15]
```diff
@@ -1,50 +1,50 @@
...
-override func loadView() {
-    view = CatalogListView(delegate: self)
+override func loadView () {
+    view = CatalogListView (delegate: self)
 }
 ```
 
 ## Provider
-Абстракция отвечающая за доступ к данным. Инкапсулирует в себе обращение к Service для получения данных из сети (или других источников) и/или DataStore, если данные должны кэшироваться.
+Abstraction responsible for accessing data. Encapsulates a call to Service to retrieve data from the network (or other sources) and/or DataStore if the data should be cached.
```
Comment (iltsarev) [inline GUIDE.md]: Provider is an abstraction responsible for accessing data. It encapsulates calls to Service to retrieve data from the network (or other sources) and/or DataStore if the data should be cached.

File: GUIDE.md [hunk 16]
```diff
@@ -1,50 +1,50 @@
...
+    static let shared = CatalogDataStore ()
 
-    var catalogModels: [CatalogObjectType: [CatalogModel]] = [:]
+    var catalogModels: [CatalogObjectType: [CatalogModel]] = [:]
 }
 ```
 
 ## ViewModel
-Содержит в себе полностью подготовленные для отображения данные для View (каждой отдельной subview). ViewModel — это DTO и не содержит в себе никакой логики.
+Contains completely prepared for display data for View (each separate subview). ViewModel is a DTO and does not contain any logic.
```
Comment (iltsarev) [inline GUIDE.md]: ViewModel contains completely prepared data for display. ViewModel is a DTO and does not contain any logic.

**Review by iltsarev — CHANGES_REQUESTED** [review]

File: GUIDE.md (hunk 14 above)
Comment (khramtsoff) [reply GUIDE.md]: ViewController is responsible for displaying the ViewModel received from Presenter ?

**Review by iltsarev — APPROVED** [review]

**Pull request merged**